napari_allencell_annotator/widgets/files_widget.py

```diff
@@ -78,8 +78,10 @@ def _handle_item_changed(self, curr_item: FileItem, prev_item: FileItem) -> None
         else:
             self._annotator_model.set_previous_image_index(self.row(prev_item))
 
-        self._annotator_model.set_curr_img_index(self.row(curr_item))
-
+        if curr_item is None:
+            self._annotator_model.set_curr_img_index(None)
+        else:
+            self._annotator_model.set_curr_img_index(self.row(curr_item))
```

Comment:
In annotation_model, curr_img_index is set to None before annotation starts. I need to check if curr_item is None to prevent it from assigning curr_img_index to -1, which is returned by self.row(None). I'm not sure if I should do this or change other parts of the code as described in my main PR comment.